### PR TITLE
Update ens-de-lyon-centre-d-ingenierie-documentaire.csl

### PR DESCRIPTION
**Pull request by ahahl**
Minor changes and improvements for some types (post, bill...)
I also had to make a workout for DOIs not to appear when pages variable exists (it would be nice if this could be included directly in Zotero's functionality concerning URLs though).
Thanks !

### CODE REVIEW

File: ens-de-lyon-centre-d-ingenierie-documentaire.csl [hunk 1]
```diff
@@ -109,7 +109,7 @@
                 <name-part name="family" font-variant="small-caps"/>
               </name>
               <et-al font-style="italic"/>
-              <label form="short" prefix="&#160;(" suffix=".)"/>
+              <label form="short" prefix=" (" suffix=".)"/>
```
Comment (rmzelle) [inline ens-de-lyon-centre-d-ingenierie-documentaire.csl]: @ahahl, are you unescaping these non-breaking spaces on purpose (or replacing them by regular spaces)?

**Comment by rmzelle** [comment]
> (it would be nice if this could be included directly in Zotero's functionality concerning URLs though)

We're actually considering asking Zotero to remove their special "don't-print-the-URL-if-there-are-pages" toggle, since it's logic can and probably should be expressed in the style itself, and because not all CSL-based reference managers have this toggle. (see https://github.com/citation-style-language/styles/issues/2495 for discussion)

File: ens-de-lyon-centre-d-ingenierie-documentaire.csl (hunk 1 above)
Comment (rmzelle) [inline ens-de-lyon-centre-d-ingenierie-documentaire.csl]: (if you aren't, could you put the escape codes back? We very easily lose unescaped special spaces, which often turn into regular spaces when we copy and paste style code around)

Comment (ahahl) [inline ens-de-lyon-centre-d-ingenierie-documentaire.csl]: Hi rmzelle, thanks for your comments. My intention is to have normal spaces here, as I see no justification for having non-breaking spaces instead.

**Comment by ahahl** [comment]
And considering the "don't-print-the-URL-if-there-are-pages" function, removing it is a solution indeed, which would avoid several problems. Although I know some users who can't edit CSL styles do like it, it is also misleading (especially as it does not manage the DOI in the same way as the URL).
About the ens-de-lyon-centre-d-ingenierie-documentaire.csl style, we are considering with coeditor smarchand creating a web version of the style (with all URL when available) and keep this one for more usual printed bibliographies.

File: ens-de-lyon-centre-d-ingenierie-documentaire.csl (hunk 1 above)
Comment (rmzelle) [inline ens-de-lyon-centre-d-ingenierie-documentaire.csl]: Okay, thanks, just making sure this was intentional.

**Comment by rmzelle** [comment]
Thanks!

> we are considering with coeditor smarchand creating a web version of the style (with all URL when available)

Note that we typically only accept CSL styles into the repository for which widespread distribution makes sense. If you make a web version for e.g. just the website of your organization, it might make more sense to distribute the style yourself as necessary.

**Comment by ahahl** [comment]
Hi again :)
It seems the update hasn't been taken into account on Zotero style repository, did I miss a step somewhere?
Sorry for troubling, and thanks for your help!

**Comment by rmzelle** [comment]
Looks like something is broken. We've pinged the Zotero devs, who usually fix stuff like this on the same day.

**Comment by rmzelle** [comment]
@ahahl, the updated style is now in the Zotero repository as well. Thanks for letting us know something was wrong.

**Pull request merged**